firmware/application/apps/ui_freqman.cpp

```diff
@@ -155,140 +149,116 @@ FrequencySaveView::FrequencySaveView(
     button_save_timestamp.on_select = [this, &nav](Button&) {
         on_save_timestamp();
     };
-
     options_category.on_change = [this, value](size_t category_id, int32_t) {
         change_category(category_id);
-        big_display.set(value);
+        big_display.set(value);  // ???
```

Comment:
Looks like an unneeded call, if big_display is already set to 'value' before calling options_category.on_change.

Reply:
Only used in 'Save' view